.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,35 @@
+name: Azion Framework adapter build and test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build-and-test:
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+        os: [ubuntu-latest, macos-latest]
+
+    name: Test - ${{ matrix.os }} - Node v${{ matrix.node-version }}
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions@github.com
```

Comment:
Do we really need this? Due to some npm dependency or something similar?

Reply:
It is necessary for testing the Init parameter. The test clones a repository and needs the git user.

Reply:
Maybe add a small one-line comment to make that explicit then